### ISSUE TITLE
Test Java client on node election stress tests.

### ISSUE THREAD

**Issue by YoEight** [EventStore/EventStoreDB-Client-Java#110]
Approving it to unblock the client, but tests around that should be tackled as a separate issue.

_Originally posted by @oskardudycz in https://github.com/EventStore/EventStoreDB-Client-Java/pull/107#pullrequestreview-795319889_

**Comment by ylorph** [comment]
Let's close doesn't look like the node client tests this anyway
